src/metabase/sync/sync_metadata/fields/sync_instances.clj

```diff
@@ -41,7 +41,7 @@
   "Insert new Field rows for for all the Fields described by `new-field-metadatas`."
   [table :- i/TableInstance, new-field-metadatas :- [i/TableMetadataField], parent-id :- common/ParentID]
   (when (seq new-field-metadatas)
-    (db/insert-many! Field
+    (t2/insert-returning-pks! Field
```

Comment:
why PKs?

Reply:
`insert-many` == `insert-returning-pks` and we do use the result of this cal.

sorry I originally put in the description that `insert-many` == `insert-returning-pks`